packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -247,35 +257,76 @@ export class MaterialPanel extends ConnectedLitElement {
     }
   }
 
+  renderVariantsTab() {
+    const hasVariants = getModelViewer().availableVariants.length > 0;
+    return html`
+    <me-expandable-tab tabName="Variants" .open=${hasVariants}>
+      <div slot="content">
+      <me-section-row
+      style='display: ${hasVariants ? '' : 'none'}'>
+        <me-dropdown
+          id="variant-selector"
+          @select=${this.onSelectVariant}
+          >${
+        getModelViewer().availableVariants.map(
+            (name, id) =>
+                html`<paper-item value="${name}">(${id}) ${name}</paper-item>`)}
+        </me-dropdown>
+      </me-section-row label>
+      <div>
+    </me-expandable-tab>
+    `;
+  }
+
   renderSelectMaterialTab() {
     return html`
     <me-expandable-tab tabName="Selected Material" .open=${true} .sticky=${
         true}>
-      <me-dropdown
-        slot="content"
-        id="material-selector"
-        @select=${this.onSelectMaterial}
-        >${
-        getModelViewer()?.model?.materials.map(
-            (material, id) => html`<paper-item value="${id}">(${id}) ${
-                material.name ? material.name :
-                                'Unnamed Material'}</paper-item>`)}
-      </me-dropdown>
+      <div slot="content">
+        <me-section-row>
```

Comment:
I think  this can just remain  unchanged from master, right? Are these divs and rows doing anything  useful? Seems to be causing unwanted centering too.

Reply:
ahhh yes